Src/DependencyCollector/Shared.Tests/Implementation/FrameworkSqlProcessingTest.cs

```diff
@@ -315,6 +185,14 @@ public void Dispose()
             Assert.IsTrue(
                 remoteDependencyTelemetryActual.Duration <= TimeSpan.FromMilliseconds(valueMax),
                 string.Format(CultureInfo.InvariantCulture, "Value (dependency duration = {0}) in the sent telemetry should not be significantly bigger than the time duration between start and end", remoteDependencyTelemetryActual.Duration));
+
+            string versonStr = Assembly.GetAssembly(typeof(DependencyTrackingTelemetryModuleTest)).GetCustomAttributes(false)
+                    .OfType<AssemblyFileVersionAttribute>()
+                    .First()
+                    .Version;
+            Version version = new Version(versonStr);
+            string expectedVersion = "rddf:" + version.ToString(3) + "-" + version.Revision;
+            Assert.AreEqual(expectedVersion, remoteDependencyTelemetryActual.Context.GetInternalContext().SdkVersion);
```

Comment:
Would it make sense to put this block of version lookup code in a helper method that can be reused by the different tests?